.github/workflows/test-m1.yml

```diff
@@ -1,50 +0,0 @@
-name: Unit-tests on M1
```

Comment:
Since that test is already on GHA, I don't think we need the grace period where we run both workflows at the same time. LMK if you disagree.